sktime/forecasting/arch/_uarch.py

```diff
@@ -464,6 +464,29 @@ def _predict_var(self, fh=None, X=None, cov=False):
 
         return pred_var
 
+    def _predict_proba(self, fh, X, marginal=True):
+        """Compute/return fully probabilistic forecasts.
+
+        Parameters
+        ----------
+        fh : int, list, np.array or ForecastingHorizon (not optional)
+            The forecasting horizon encoding the time stamps to forecast at.
+            if has not been passed in fit, must be passed, not optional
+        X : sktime time series object, optional (default=None)
+            Exogeneous time series for the forecast
+        marginal : bool, optional (default=True)
+            whether returned distribution is marginal by time index
+
+        Returns
+        -------
+        pred_dist : sktime BaseDistribution
+            predictive distribution
+        """
+        pred_dist = super()._predict_proba(fh=fh, X=X, marginal=marginal)
```

Comment:
why is this needed?

Reply:
but why does the `super` not automatically take care of it?

Reply:
`super` takes care of it. I should have looked into that closely.